### ISSUE TITLE
Output file name with encoding tags like AV1.SVT.CRF28.SP8.Opus

### ISSUE THREAD

**Issue by DejayRezme** [n00mkrad/nmkoder#10]
Just a suggestion, not sure if anybody else would like better output file naming. For example:

*"My.Stories.S01E05.1080p.x265.AAC.mkv"* becomes *"My.Stories.S01E05.1080p.AV1.SVT.CRF28.SP8.FG16.Opus.mkv"*

CRF, speed/preset, film grain and encoder are saved which can be handy for remembering later. If film grain is 0 the tag is omitted. 

Tags like x264, h264, H.264, x265, h265, H.265, HEVC, DD5.1 could be removed. I'd also like the option to convert space into "." and things like ".-." or "_-_" into a "." but that might just be me.

There could be a button with a "drop down checkbox menu" next to where you choose the extension in NMKoder.

For comparison, fastflix appends something like "-fastflix-9483". VidCoder appends "(Encoded)"

For background, I've made a crude command line tool for myself that launches ffmpeg and svt and accepts wildcards to run through multiple files in a folder and then puts the encoding parameters into the file name in front of the extension. It's also handy because it can skip files where the output file already exists for those parameters or the input file contains av1 when I rerun the same command to continue.

**Comment by n00mkrad** [comment]
Good idea. I think removing old suffixes will be too complicated, but adding the encoder and settings is a great idea. I've beed doing that manually anyway.
